Base/Base.Test/FileLoggerTests.cpp

```diff
@@ -43,7 +43,7 @@ std::string getCurrentDate()
     return std::string(buf);
 }
 
-void check()
+void waitThreadTerminate()
```

Comment:
This is misleading: I would expect it to do something like { while(thread.notTerminated()) wait(); }
And this just waits